client/my-sites/theme/main.jsx

```diff
@@ -348,46 +348,58 @@ const ThemeSheet = React.createClass( {
 		const analyticsPageTitle = `Themes > Details Sheet${ section ? ' > ' + titlecase( section ) : '' }${ siteID ? ' > Site' : '' }`;
 
 		const { name: themeName, description } = this.props;
-		const title = i18n.translate( '%(themeName)s Theme', {
+		const title = themeName && i18n.translate( '%(themeName)s Theme', {
```

Comment:
> What was the reason for the `themeName &&` change?

Basically to retain logic that was previously [here](https://github.com/Automattic/wp-calypso/pull/7600/files#diff-f576b3c13daa17f9ccee514c4c46bc07L359).